server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/json/EmailGetSerializer.scala

```diff
@@ -92,19 +93,43 @@ object EmailGetSerializer {
             "subParts" -> part.subParts.map(list => list.map(writes)))))
     }
 
-  private def emailWritesWithPropertyFilter(properties: Properties)(implicit partsWrites: Writes[EmailBodyPart]): Writes[Email] = {
-    implicit val emailMetadataWrites: Writes[EmailMetadata] = Json.writes[EmailMetadata]
+  private def emailWritesWithPropertyFilter(properties: Properties)(implicit partsWrites: Writes[EmailBodyPart]): Writes[EmailView] = {
+    implicit val emailMetadataWrites: OWrites[EmailMetadata] = ((JsPath \ "id").write[MessageId] and
```

Comment:
Why do you write the writes by hand ? 
Json.writes[...] will return an OWrites if it is what you want ?
whereas Json.valueWrites will return a Writes

Reply:
> Json.writes[...] will return an OWrites if it is what you want ?

I tried that.... Returned me a Writes and not a OWrites

Reply:
My bad